lib/src/native/realm_core.dart

```diff
@@ -2219,7 +2219,7 @@ class _RealmCore {
   String getAppDirectory() {
     try {
       if (!isFlutterPlatform || Platform.environment.containsKey('FLUTTER_TEST')) {
-        return Directory.current.absolute.path; // dart or flutter test
+        return Directory.systemTemp; // dart or flutter test
```

Comment:
This is a breaking change since it moves database files in dart apps (e.g. console apps) to systemTemp, which is not what we want.

Reply:
From what I can see, `getAppDirectory` is a private API (not documented in dartdoc) and it's only caller is `Configuration.defaultSotragePath` when `isFlutterPlatform` is true. Maybe this could be refactored to be more obvious.

Since this path is not documented anywhere, I think it should be OK to change it for Dart unit tests and Flutter tests?